CHANGELOG.md

```diff
@@ -1,9 +1,17 @@
 # Changelog
 
+## HEAD
+
+- Breaking: remove support for Node 8.
+- Change base `ecmaVersion` from `2018` to `2020`.
```

Comment:
Heads up that ecmaVersion 2020 requires eslint version >= 6

Reply:
Thanks, added it to the changelog: https://github.com/mapbox/eslint-config-mapbox/commit/4ea168ba1ccb1a84df1357ceb006fd0b64fb0990